config/console/console_init.yaml

```diff
@@ -22,30 +22,14 @@ spec:
             - containerPort: 9001
               protocol: TCP
           args:
-            - "--ssl --cert=/var/serving-cert/tls.crt --key=/var/serving-cert/tls.key"
+            - "--ssl"
+            - "--cert=/var/serving-cert/tls.crt"
+            - "--key=/var/serving-cert/tls.key"
```

Comment:
This should be a separate commit

Reply:
It's a trivial change. Let's keep it as part of it.

Reply:
It is precisely because it is a trivial change that it should be moved out. :) Keep the commit tight and focused on only the relevant logical changes.

Reply:
So should I create a different back-port for this?